### PR TITLE
docs(community): add section for community strategies with netrc as the first entry

### PR DESCRIPTION
**Pull request by travi**

### CODE REVIEW

**Comment by travi** [comment]
here's a first stab at adding a community strategy section. i also grouped the official strategies to match, but please highlight anything that you'd like to see done differently and i can adjust further

File: README.md [hunk 1]
```diff
@@ -426,6 +428,23 @@ const { token } = await auth();
...
+
+### .netrc authentication
+
+Similar to [token authentication](#token-authentication), but reads the token from your `~/.netrc` file
+
+Example
+
+```js
+// expects a personal access token to be set as `login` in the `~/.netrc` file for `api.github.com`
+const createNetrcAuth = require('octokit-netrc-auth').createNetrcAuth;
```
Comment (gr2m) [inline README.md:441]: I think I would use deconstruction here, as I'm using it in all Octokit examples

```suggestion
const { createNetrcAuth } = require('octokit-netrc-auth');
```

File: README.md [hunk 2]
```diff
@@ -426,6 +428,23 @@ const { token } = await auth();
...
+const auth = createNetrcAuth();
+const {token} = await auth();
```
Comment (gr2m) [inline README.md:443]: I use the default `prettier` formatting in all octokit project

```suggestion
const { token } = await auth();
```

File: README.md (hunk 1 above)
Comment (travi) [reply README.md:441]: Yep good call. I didn't pay close enough attention when I was trying to match.

File: README.md (hunk 2 above)
Comment (travi) [reply README.md:443]: Fair :)

**Pull request merged**